tests/test_data_transformers.py

```diff
@@ -26,7 +26,7 @@ class DataTransformerTest:
         expected_multiple_outputs: The expected outputs for the multiple inputs.
     """
 
-    def __init__( # noqa: D107
+    def __init__(  # noqa: D107
```

Comment:
what does # noqa: D107 mean ?

Reply:
don;t know, igor put it
but it is probably a line that is used to remove a given from certain linting checking